test/unit/finance/bill_variable_for_plan_changed_test.rb

```diff
@@ -16,7 +18,7 @@ class Finance::NoVariableCostClass < ActiveSupport::TestCase
   end
 
   test "bill for variable" do
-    Timecop.travel(15.days.ago) if Time.now.mday == 1
+    Timecop.travel(15.days.ago) if Time.now.mday == 1 # rubocop:disable Rails/TimeZone we don't use timezones in billing
```

Comment:
```suggestion
    # TODO: there is an issue not covered when the date is the 1st day of the month, see issue (???)
    Timecop.travel(Date.parse('2021-11-15')) # rubocop:disable Rails/TimeZone we don't use timezones in billing
```

Reply:
I am not changing any logic as per @akostadinov's request. This specific thing I address it here https://github.com/3scale/porta/pull/2841/files#diff-9da0c59a29bd8678f080d4d1fe561f4865eb16c8d3cbfd3c0dd2c83ccf6d9851R24-R25